### DISCUSSION TITLE
Test request ignores servers and baseServerURL

### DISCUSSION THREAD

**Discussion by crukundo** [scalar/scalar#3475] [Q&A]
Hello, Here's my sample config:

"@scalar/api-reference": "^1.24.8",
"@scalar/api-reference-react": "^0.3.6",

const config: ReferenceConfiguration = {
  theme: "none",
  layout: "modern",
  baseServerURL: "https://sandbox.api.sample",
  spec: {
    url: "https://sandbox.api.sample/docs/openapi.json",
  },
  hideModels: false,
  servers: [
    { url: "https://sandbox.api.sample", description: "Sandbox API" },
    { url: "https://api.splice.sample", description: "Production API" },
  ],
};

What's confusing me is every time you try to use the Test Request client, it uses the current base domain where I'm hosting my api reference: docs.sample.com so all my requests are going through `docs.sample.com` for example: `/health_check` get endpoint will use `docs.sample.com/health_check` rather than say  `sandbox.api.sample/health_check` or one of the Server URLS I specified. I tried to use `baseServerURL` but that doesn't do it either. 

Is there anything I'm doing wrong? Please advise.

**Comment by hanspagel** [comment]
That’s not you, that’s a bug! Thanks for the report, I’ve created an issue here: https://github.com/scalar/scalar/issues/3476

**Reply by crukundo** [reply]
Thanks for the quick response. Following #3476 closely.

**Reply by amritk** [reply]
@crukundo the baseServerURL part has been fixed in https://github.com/scalar/scalar/pull/3776. Working on servers now 🫡
